README.md

```diff
@@ -9,23 +9,29 @@ configures the Docker client to talk to them.
 It works a bit like this:
 
 ```console
-$ docker-machine create -d virtualbox dev
-Creating CA: /home/username/.docker/machine/certs/ca.pem
-Creating client certificate: /home/username/.docker/machine/certs/cert.pem
-Image cache does not exist, creating it at /home/username/.docker/machine/cache...
-No default boot2docker iso found locally, downloading the latest release...
-Downloading https://github.com/boot2docker/boot2docker/releases/download/v1.6.2/boot2docker.iso to /home/username/.docker/machine/cache/boot2docker.iso...
-Creating VirtualBox VM...
-Creating SSH key...
-Starting VirtualBox VM...
-Starting VM...
-To see how to connect Docker to this machine, run: docker-machine env dev
+$ docker-machine create -d virtualbox default
```

Comment:
Updated this `create` output in several places to be more current and also to favor `default` over `dev`.